releasenotes/2.1.1.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.1
+date: 2018-08-09 16:00:00
+release_notes: true
+permalink: /releasenotes/2.1.1/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.1 is the first bugfix release after jclouds 2.1 and comes with several bugfixes and performance improvements. 
+You can [read the official announcement here](https://s.apache.org/jclouds211). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.1 new features include:
+
+* Add support for c5 and R4 instance types in AWS ec2.
```

Comment:
[minor] "EC2"? And `c5` and `R4` either both in upper- or lowercase, or are they indeed in different cases in the AWS docs?

Reply:
Also m5 instance types were added but the JIRA (https://issues.apache.org/jira/browse/JCLOUDS-1436) fixVersion was wrong. I just updated it to 2.1.1
@demobox AWS docs use uppercase or lowercase, we should keep all lowercase imho